firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHandler.java

```diff
@@ -86,9 +86,7 @@ private synchronized void beginRealtime() {
   public synchronized ConfigUpdateListenerRegistration addRealtimeConfigUpdateListener(
       @NonNull ConfigUpdateListener configUpdateListener) {
     listeners.add(configUpdateListener);
-    if (canCreateRealtimeHttpClientTask()) {
```

Comment:
Is `canCreateRealtimeHttpClientTask` used anymore? Maybe delete it if not

Reply:
Oh yeah it's not, removed. Thanks!